destination.go

```diff
@@ -343,22 +281,51 @@ func (d *Destination) formatColumnsAndValues(ctx context.Context, table string,
 	return colArgs, valArgs, nil
 }
 
-// getKeyColumnName will return the name of the first item in the key or the
-// connector-configured default name of the key column name.
-func (d *Destination) getKeyColumnName(key opencdc.StructuredData, defaultKeyName string) string {
-	if len(key) > 1 {
-		// Go maps aren't order preserving, so anything over len 1 will have
-		// non-deterministic results until we handle composite keys.
-		panic("composite keys not yet supported")
+func (d *Destination) hasKey(e opencdc.Record) bool {
+	structuredKey, ok := e.Key.(opencdc.StructuredData)
+	if !ok {
+		return false
 	}
-	for k := range key {
-		return k
+	return len(structuredKey) > 0
+}
+
+// cleanRecord makes sure the record key and payload are structured data.
+func (d *Destination) cleanRecord(r opencdc.Record) (opencdc.Record, error) {
```

Comment:
May I suggest maybe naming this method `ensureStructuredRecord`?

Reply:
You may 🙇